ui/app/components/sidebar/nav/access.hbs

```diff
@@ -3,7 +3,13 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-<Hds::SideNav::Portal @ariaLabel="Access Navigation Links" data-test-sidebar-nav-panel="Access" as |Nav|>
+<Hds::SideNav::Portal
+  @ariaLabel="Access Navigation Links"
```

Comment:
required to have tabindex and recommended role for a11y testing. only an issue on firefox and not on chrome

Reply:
😵 We can't win!